src/pages/index.js

```diff
@@ -504,7 +504,10 @@ export default function App() {
             </div>
 
             <div className={styles.spacer} />
-            <h1 className={styles.h1}>Roadmap</h1>
+
+            <h1 className={styles.h1} id="roadmap">
```

Comment:
```suggestion
            <h1 className={styles.h1, "anchor-with-padding"} id="roadmap">
```

This _should_ make it where the navbar doesn't obstruct the view when you are linked to it